pkg/cmd/cmd.go

```diff
@@ -145,51 +217,13 @@ func SetupRootCommand(cmd *cobra.Command) {
 	cmd.PersistentFlags().BoolP("help", "", false, "help for daytona")
 	cmd.Flags().BoolP("version", "v", false, "Display the version of Daytona")
 
-	var telemetryService telemetry.TelemetryService
-	clientId := config.GetClientId()
-	telemetryEnabled := config.TelemetryEnabled()
-
-	if telemetryEnabled {
-		telemetryService = posthogservice.NewTelemetryService(posthogservice.PosthogServiceConfig{
-			ApiKey:   internal.PosthogApiKey,
-			Endpoint: internal.PosthogEndpoint,
-		})
-	}
-
-	startTime := time.Now()
-
 	cmd.PreRun = func(cmd *cobra.Command, args []string) {
 		versionFlag, _ := cmd.Flags().GetBool("version")
 		if versionFlag {
 			versionCmd.Run(cmd, []string{})
 			os.Exit(0)
 		}
 	}
-
-	cmd.PersistentPreRun = func(cmd *cobra.Command, args []string) {
-		if telemetryService != nil {
-			err := telemetryService.TrackCliEvent(telemetry.CliEventCmdStart, clientId, getCmdTelemetryData(cmd))
-			if err != nil {
-				log.Error(err)
-			}
-		}
-	}
-
-	cmd.PersistentPostRun = func(cmd *cobra.Command, args []string) {
-		endTime := time.Now()
-
-		if telemetryService != nil {
-			defer telemetryService.Close()
-			execTime := endTime.Sub(startTime)
-			props := getCmdTelemetryData(cmd)
-			props["exec time (µs)"] = execTime.Microseconds()
-
-			err := telemetryService.TrackCliEvent(telemetry.CliEventCmdEnd, clientId, props)
-			if err != nil {
-				log.Error(err)
-			}
-		}
-	}
 }
 
 func RunInitialScreenFlow(cmd *cobra.Command, args []string) {
```

Comment:
The "`.Run`" functions of all commands in here are undefined leading to panics

![image](https://github.com/user-attachments/assets/fa18c0c1-fccd-4070-a878-bb95ec9d50d7)

![image](https://github.com/user-attachments/assets/20c9f2d2-2e1b-4f5d-aa64-0452eaba142a)